schedules/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "temporal-schedules",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "lint": "eslint .",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "schedule.start": "ts-node src/start-schedule.ts",
+    "schedule.go-faster": "ts-node src/go-faster.ts",
+    "schedule.pause": "ts-node src/pause-schedule.ts",
+    "schedule.unpause": "ts-node src/unpause-schedule.ts",
+    "schedule.delete": "ts-node src/delete-schedule.ts"
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
+  "dependencies": {
+    "@temporalio/activity": "^1.0.0",
```

Comment:
These should be "^1.5.2"

Reply:
Why did this change? I remember changing everything to `^1.0.0` at Roey's suggestion

Reply:
Do you remember @bergundy argument for this suggestion?

IMHO: With `^1.0.0`, NPM will happily use whatever old version that the user has in his NPM cache. Many examples could therefore fail to run in user environment. Keeping version number in samples up-to-date means we silently push users toward updating. My intention would be to automatically update this number from sdk-typescript's release CI.

Reply:
Probably so we didn't have to keep doing the manual script. If it's automatic, sounds good!